front/app/containers/Admin/pages/EditPageForm.tsx

```diff
@@ -84,14 +81,14 @@ const EditPageForm = ({ params: { pageId } }: Props & WithRouterProps) => {
     clHistory.push('/admin/pages');
   };
 
-  const renderFn = (pageId: string, slug: string) => (
+  const renderFn = (pageId: string, code: TPageCode) => (
     props: FormikProps<FormValues>
   ) => {
     return (
       <PageForm
         {...props}
         pageId={pageId}
-        hideSlugInput={[...FIXED_PAGES, ...POLICY_PAGES].includes(slug)}
+        hideSlugInput={EDITABLE_SLUG_PAGES.has(code)}
```

Comment:
Removed this bit of code. Turns out the slug input is always hidden, no matter what value you give to the prop- so something broken in the `PageForm` component it seems. Will take this up in next iteration

Reply:
It's hidden when we're creating a new page and shown if we're in the edit page form if I remember correctly. For some pages, it may never show, not even in the edit form.